capa/render/utils.py

```diff
@@ -56,6 +58,18 @@ def capability_rules(doc: rd.ResultDocument) -> Iterator[rd.RuleMatches]:
         yield rule
 
 
+def load_rules_prevalence() -> Dict[str, str]:
+    CD = Path(__file__).resolve().parent.parent.parent
+    file = CD / "assets/rules_prevalence_data/rules_prevalence.json.gz"
```

Comment:
This can be updated later when `get_default_root` logic is moved to another module.

Reply:
Do we have an issue for this already? If not, could you please create one?

Reply:
There was some discussion in PR 1820 regarding breaking off such logic froom `capa.main` to `capa.loader`

Reply:
never mind, I meant https://github.com/mandiant/capa/issues/1818